client/elements/text/sc-segmented-text.html

```diff
@@ -293,10 +295,18 @@ <h3>Could not find the sutta text.</h3>
                   case 'sidebyside':
                       textContent.classList.add('side-by-side');
                       this._addSecondaryText();
+                      this._toggleSegmentedTextualInfo(true);
+                      requestAnimationFrame(() => {
+//                          this._addSegmentedTextualInfoTopMargins();
```

Comment:
?

Reply:
work in progress. I pushed it so you can work on the stuff we discussed.